content/file-and-directory-naming/general/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+_db_id: 768
+content_type: topic
+ready: true
+title: File and directory naming
+---
+
+## Naming files and directories
+
+People often develop a habit of writing messy and unorganized code initially, only to attempt to improve the naming conventions after the code functions. This practice is problematic as it can introduce errors, and developers may overlook important details. It is far more advantageous to cultivate good coding habits from the outset. While it may seem slower initially, you will ultimately achieve greater efficiency once you become accustomed to it.
+
+You can organize your files and directories in a logical and consistent manner by using naming conventions. The use of naming conventions helps to ensure the correct information can be located, identified, and retrieved from a file system quickly, as well as that it is stored in a secure location.
```

Comment:
This paragraph is not related to the one above. Should it be in the same section? 

Maybe there should be a heading that says "## A common pitfall" or something like that at the bottom that tells people about the common problem.

And then it would be good to say something about why naming matters near the top somewhere.